embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -44,11 +44,38 @@ while true; do
     esac
 done
 
+embulk_args="$@"
+
+while [ $# -gt 0 ] ; do
+    case "$1" in
+        "-b" | "--bundle")
+            shift
+            export EMBULK_BUNDLE_PATH="$1"
+            shift
+            break
+            ;;
+        *)
+            shift
+            ;;
+    esac
+done
+
+if test -z ${EMBULK_BUNDLE_PATH}; then
+    unset EMBULK_BUNDLE_PATH
+    unset BUNDLE_GEMFILE
+    export GEM_HOME="$(cd && pwd)/.embulk/jruby/$(java -cp $0 org.jruby.Main -e 'print RbConfig::CONFIG["ruby_version"]')"
```

Comment:
That may be worth discussing, but `$()` is now POSIX-compliant. Recent `sh` like `ash` has no problems with that. Although the really legacy `sh` may have problems with `$()`, I believe we have no problems with recent `sh` (not `bash`). It may be still worth choosing the legacy option, though.
http://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03

The reason of `cd && pwd` was that it was originally `java.lang.System.properties["user.home"] || ENV['HOME']` in JRuby, which might be to support irregular cases like Windows. I thought it may be fine to use `$HOME` since this is the `.sh` part (non-Windows), but I was not very sure about how it works with Bash on Windows or else like that.

Will fix the `export` one.

I guess `unset` for environment variables is fine in `sh`.